modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_convertToOIDC.toml

```diff
@@ -1,8 +1,7 @@
 [server]
```

Comment:
rename the file, word separator is not consistant

Reply:
Fixed.